.github/workflows/build_and_test.yml

```diff
@@ -88,74 +93,57 @@ jobs:
-      with:
-        user: __token__
-        password: ${{ secrets.PYPI_PYTKET_QUEST_API_TOKEN }}
-        verbose: true
+      - name: Download all wheels
+        # downloading all three files into the wheelhouse
+        # all files are identical, so there will only be one file
+        uses: actions/download-artifact@v4
+        with:
+          path: wheelhouse
+          pattern: artefact-*
+          merge-multiple: true
+      - name: Put them all in the dist folder
+        run: |
+          mkdir dist
+          for w in `find wheelhouse/ -type f -name "*.whl"` ; do cp $w dist/ ; done
+      - name: Publish wheels
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_PYTKET_QUEST_API_TOKEN }}
+          verbose: true
 
   build_docs:
     name: Build docs
     if: github.event_name == 'release'
     needs: publish_to_pypi
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v4
-      with:
-        fetch-depth: '0'
-    - name: Set up Python 3.11
-      uses: actions/setup-python@v5
-      with:
-        python-version: '3.11'
-    - name: Download all wheels
-      uses: actions/download-artifact@v4
-      with:
-        path: wheelhouse
-    - name: Install pip, wheel
-      run: pip install -U pip wheel
-    - name: Install extension
-      run: for w in `find wheelhouse/ -type f -name "*.whl"` ; do pip install $w ; done
-    - name: Install docs dependencies
-      run: |
-        pip install -r .github/workflows/docs/requirements.txt
-    - name: Build docs
-      timeout-minutes: 20
-      run: |
-        cd .github/workflows/docs
-        mkdir extensions
-        ./build-docs -d ${GITHUB_WORKSPACE}/.github/workflows/docs/extensions/api
-    - name: Upload docs as artefact
-      uses: actions/upload-pages-artifact@v3
-      with:
-        path: .github/workflows/docs/extensions
-  
-  publish_docs:
-    name: Publish docs
-    if: github.event_name == 'release'
-    needs: build_docs
-    runs-on: ubuntu-22.04
-    permissions:
-      pages: write
-      id-token: write
-    environment:
-      name: github-pages
-      url: ${{ steps.deployment.outputs.page_url }}
-    steps:
-    - name: Deploy to GitHub Pages
-      id: deployment
-      uses: actions/deploy-pages@v4.0.4
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: "0"
+          submosules: true
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+      - name: Download all wheels
+        uses: actions/download-artifact@v4
+        with:
+          path: wheelhouse
+      - name: Install pip, wheel
+        run: pip install -U pip wheel
+      - name: Install poetry
+        run: pip install poetry
+      - name: Install extension
+        run: for w in `find wheelhouse/ -type f -name "*.whl"` ; do poetry install $w ; done
```

Comment:
Ohh actually I think you might need to do `poetry run pip install` for the wheel rather than `poetry instal` here